Patches/SabotageSystemPatch.cs

```diff
@@ -97,6 +138,64 @@ public static void Initialize()
         modifiedCooldownSec = Options.SabotageCooldown.GetFloat();
     }
 
+    public static bool Prefix(SabotageSystemType __instance, [HarmonyArgument(0)] PlayerControl player, [HarmonyArgument(1)] MessageReader msgReader)
+    {
+        var newReader = MessageReader.Get(msgReader);
+        var amount = newReader.ReadByte();
+        var nextSabotage = (SystemTypes)amount;
+
+        if (Options.DisableSabotage.GetBool())
+        {
+            return false;
+        }
+
+        Logger.Info("Sabotage" + ", PlayerName: " + player.GetNameWithRole() + ", SabotageType: " + nextSabotage.ToString(), "RepairSystem");
+
+        return CanSabotage(player, nextSabotage);
+    }
+    private static bool CanSabotage(PlayerControl player, SystemTypes systemType)
+    {
+        var playerRole = player.GetCustomRole();
+
+        if (systemType == SystemTypes.Comms)
+        {
+            if (playerRole.Is(CustomRoles.Camouflager) && !Camouflager.CanUseCommsSabotage.GetBool()) 
+                return false;
+        }
+
+        if (player.Is(CustomRoleTypes.Impostor) && (player.IsAlive() || !Options.DeadImpCantSabotage.GetBool())) return true;
+
+        switch (playerRole)
+        {
+            case CustomRoles.Jackal when Jackal.CanUseSabotage.GetBool():
+                return true;
+
+            case CustomRoles.Sidekick when Jackal.CanUseSabotageSK.GetBool():
+                return true;
+
+            case CustomRoles.Bandit when Bandit.CanUseSabotage.GetBool():
+                return true;
+
+            case CustomRoles.Glitch:
+                Glitch.Mimic(player);
+                return true;
```

Comment:
return false here cause the sabotage is just a trigger for mimic, Glitch cannot actually sabotage

Reply:
Fixed